src/ralph/networks/admin.py

```diff
@@ -45,11 +48,42 @@ class Meta:
 
 
 @register(Network)
-class NetworkAdmin(RalphMPTTAdmin):
+class NetworkAdmin(RalphAdmin):
+
+    def subnetwork(self, obj):
+        count = obj.get_descendant_count()
+        if count:
+            return '<a href="{}?{}">{} ({})</a>'.format(
+                reverse('admin:networks_network_changelist'),
+                urlencode({'parent': obj.pk}),
+                _('Show'),
+                count
+            )
+        return ''
+
+    subnetwork.short_description = 'Subnetwork'
+    subnetwork.allow_tags = True
+
+    def name(self, obj):
```

Comment:
do we really need this field declaration? why default-way is not working (just declaring `'name'` on `list_display`)?